### ISSUE TITLE
sclang: cmd+w doesn't close child windows

### ISSUE THREAD

**Issue by htor** [supercollider/supercollider#4309]
hitting cmd+w on a child window like a plot or a freq scope doesn't close it. it should close the window.

<img width="512" alt="image" src="https://user-images.githubusercontent.com/1709263/52861159-e82e0780-3131-11e9-885e-85919cd012a6.png">

## Reproduce

```js
{ SinOsc.ar }.plot   // window opens
```

try close the window with `cmd+w`

## Environment

* SuperCollider version: 3.10.2 Built from branch 'HEAD' [834c036d]
* Operating system: macOS 10.13.6 (17G5019)

**Comment by dyfer** [comment]
This happens to me too on macOS (10.13.6).
Also, I wonder if https://github.com/supercollider/supercollider/issues/3824 is related? (Question to devs I guess)

**Comment by telephon** [comment]
This window is a window in the sclang app, not in the ide. With the new Menu system, maybe it could be easily implemented - why not try it?

(see MainMenu help)

**Comment by adcxyz** [comment]
In SC 3.10.2, macOS 10.13.6, 
Cmd-W does not close any windows in sclang, no matter how you make them:
```
w = Window.new.front // does not close with Cmd-W
UserView.new.front // implicit window - also does not close with Cmd-W
```

**Comment by htor** [comment]
not sure, but will https://github.com/supercollider/supercollider/pull/4533 solve this issue?

**Comment by dyfer** [comment]
> not sure, but will #4533 solve this issue?

Unfortunately no.

**Comment by dantepfer** [comment]
Just want to add my voice to the chorus: I agree that cmd-w should close child windows in SC on OSX.

**Comment by brianlheim** [comment]
fixed in #4821, thanks @dyfer !
